snuba/datasets/storages/sessions.py

```diff
@@ -122,7 +122,10 @@ def process_query(self, query: Query, request_settings: RequestSettings) -> None
     query_processors=[MinuteResolutionProcessor(), TableRateLimit()],
     mandatory_condition_checkers=[OrgIdEnforcer(), ProjectIdEnforcer()],
     stream_loader=build_kafka_stream_loader_from_settings(
-        processor=SessionsProcessor(), default_topic=Topic.SESSIONS,
+        processor=SessionsProcessor(),
+        default_topic=Topic.SESSIONS,
+        commit_log_topic=Topic.SESSIONS_COMMIT_LOG,
+        subscription_result_topic=Topic.SUBSCRIPTION_RESULTS_SESSIONS,
```

Comment:
I think we discussed these being behind a feature flag, is that still the plan?

Reply:
(to clarify, if the base branch is master then we would want the flag, otherwise all good)

Reply:
Maybe I will just add a feature flag then.. I am not familiar with how feature flags are implemented in snuba.. do you mind pointing me to an example?